lib/matplotlib/axes/_axes.py

```diff
@@ -2604,6 +2605,136 @@ def barh(self, y, width, height=0.8, left=None, *, align="center",
                            align=align, **kwargs)
         return patches
 
+    def blabel(self, container, captions=[], fmt="%g", mode="edge",
```

Comment:
While this is in resemblance to `clabel` for contours, I'd prefer `bar_label()` as being more self-explanatory.

Reply:
Sure, no problem.